src/main/java/com/puppycrawl/tools/checkstyle/checks/DescendantTokenCheck.java

```diff
@@ -659,7 +659,7 @@ public class DescendantTokenCheck extends AbstractCheck {
     private boolean sumTokenCounts;
     /** Specify set of tokens with limited occurrences as descendants. */
     @XdocsPropertyType(PropertyType.TOKEN_ARRAY)
-    private int[] limitedTokens = CommonUtil.EMPTY_INT_ARRAY;
+    private int[] limitedTokens;
```

Comment:
@Kevin222004 Won't this throw NPE when there is no mention of this property in the config? `limitedTokens` is just not there in the config. In that case `counts` will be null and `Arrays.fill(counts, 0);` will throw an NPE.

Reply:
```diff
/*
DescendantToken
tokens = LITERAL_SWITCH
maximumDepth = 2
- limitedTokens =
minimumNumber = 1

*/
```
In this type of config, an NPE would be thrown. Is `limitedTokens` mandatory for the user to set?

Reply:
I am not sure where things have gone wrong from my side. but done

Test Added